content/doc/book/installing/kubernetes.adoc

```diff
@@ -255,6 +271,11 @@ spec:
 In this Jenkins Kubernetes deployment we have used the following:
 
 [arabic]
+. The Jenkins container version is referenced in the example YAML above by a floating tag `lts`, so the optional `imagePullPolicy: Always` line is added to pull and deploy a new LTS image release (if exists) *whenever* the pod is restarted. Otherwise, the currently deployed version is used after restart (and there's no other way to tell Kubernetes to pull a newer image, short of locating and deleting the old one from the node's cache).
+  * Note that "whenever the pod is restarted" includes e.g. server reboot or the restart requested in Web-UI to apply downloaded plugin updates.
```

Comment:
```suggestion
  * Note that "whenever the pod is restarted" includes events such as a server reboot or the restart requested in Web-UI to apply downloaded plugin updates.
```
Small syntax suggestion to use "such as" instead of "e.g.".